### DISCUSSION TITLE
Another & Fastest (Delivery Date & Time Slots)

### DISCUSSION THREAD

**Discussion by devomman** [inspireui/support#5630] [New Feature/Ideas Requests]
Dear InspireUI Team,

Kindly please add this Features for Delivery Date Plugin for us https://codecanyon.net/item/woocommerce-delivery/26548021 Because Other Delivery Date is Not Working Properly Some Times and Very High Loading Time but this plugin is very good and faster. 
So Please Integrate this to WooCommerce Please. Thanks

Best Regards

**Comment by genuinesourcecode** [comment]
Agreed

**Comment by devomman** [comment]
Dear,
Kindly please add this https://codecanyon.net/item/woocommerce-delivery/26548021 plugin support for **_delivery date & time_** slot because other plugin is very heavy and giving many bugs.

**Comment by devomman** [comment]
@minhcasi @luyx2412 @vumanhsn97 Kindly add this delivery date https://codecanyon.net/item/woocommerce-delivery/26548021 plugin api to fluxstore app because other delivery date is broken the webiste and website getting very slow.
